docs/docs/scanner/misconfiguration/custom/index.md

```diff
@@ -103,7 +103,7 @@ Any package prefixes such as `main` and `user` are allowed.
 ### Metadata
 Metadata helps enrich Trivy's scan results with useful information.
```

Comment:
Thank for this. I know it's unrelated to your change, but this sentence popped out to me. it makes metadata seem like an optional addition and also fails to explain the purpose. so while we're here could you please fix it to:

```
The policy must contain a [Rego Metadata](https://www.openpolicyagent.org/docs/latest/policy-language/#metadata) section. Trivy uses standard rego metadata to define the new policy and general information about it.
```

I used policy in accordance with the rest of the doc, when we rename to check we can do it at once.

Reply:
Done https://github.com/aquasecurity/trivy/pull/6480/commits/0757a4ffd66bfac238861b5c758caed39c963d7b